### ISSUE TITLE
TextEditor.onDidChangeScrollTop is deprecated.

### ISSUE THREAD

**Issue by Nebulosar** [georgebrindeiro/scroll-sync#25]
This is now a view method. Call TextEditorElement::onDidChangeScrollTop instead.
```
TextEditor.onDidChangeScrollTop (<embedded>:2945:18)
<unknown> (C:\Users\Harm\.atom\packages\scroll-sync\lib\scroll-sync.coffee:120:57)
ScrlSync.startTracking (C:\Users\Harm\.atom\packages\scroll-sync\lib\scroll-sync.coffee:124:17)
atom-workspace.subToggle.add.atom.commands.add.scroll-sync:toggle (C:\Users\Harm\.atom\packages\scroll-sync\lib\scroll-sync.coffee:31:28)
CommandRegistry.handleCommandEvent (<embedded>:18904:35)
handleCommandEvent (<embedded>:18642:65)
```

Scrolling down only works half of the times now

**Comment by georgebrindeiro-vtex** [comment]
@Nebulosar Thanks for reporting! I just took over this repository and got in vacations from my regular job. Seems like this issue has been reported/fixed already, just not pushed to the atom package repository. Will look into it as soon as I have figured out the update workflow.

Also, something seems to be up with my GitHub notifications, so sorry for taking a while to respond.

**Comment by georgebrindeiro-vtex** [comment]
@Nebulosar I just released version 0.2.3, check it out and see if it works now!

**Comment by Nebulosar** [comment]
Hi, thanks for looking into this!

Now works perfect again!
No need to appologize for taking the time, I can understand it when you aren't on github 100% of your time ;)
